packages/components/pie-divider/test/visual/pie-divider.spec.ts

```diff
@@ -4,7 +4,7 @@ import percySnapshot from '@percy/playwright';
 import { PieDivider, DividerProps } from '@/index';
 
 test.describe('PieDivider - Visual tests`', () => {
-    test('should display the PieDivider component successfully', async ({ page, mount }) => {
+    test.skip('should display the PieDivider component successfully', async ({ page, mount }) => {
```

Comment:
I think you can move this one level up to the describe so the whole thing skips?

Reply:
done